doozerlib/rpmcfg.py

```diff
@@ -58,8 +58,8 @@ def __init__(self, runtime, data_obj, clone_source=True,
         self.extra_os_git_vars = {}
 
         if clone_source:
-            self.source_path = self.runtime.resolve_source('rpm_{}'.format(self.rpm_name), self)
-            self.source_head = self.runtime.resolve_source_head('rpm_{}'.format(self.rpm_name), self)
```

Comment:
this is the only place where `parent` diverged from `self.name`. I feel like that must have been significant at some point but can't think why it would be.